src/debugger/scriptImporter.ts

```diff
@@ -71,7 +71,7 @@ export class ScriptImporter {
      * Writes the script file to the project temporary location.
      */
     private writeAppScript(scriptBody: string, scriptUrl: url.Url): Q.Promise<String> {
-        let scriptFilePath = path.join(this.sourcesStoragePath, scriptUrl.pathname); // scriptFilePath = "$TMPDIR/index.ios.bundle"
+        let scriptFilePath = path.join(this.sourcesStoragePath, path.basename(scriptUrl.pathname)); // scriptFilePath = "$TMPDIR/index.ios.bundle"
```

Comment:
@MSLaguana I hate this timezone difference :-) The React Native packager can serve multiple apps at a time, but vscode won't hande multiple apps concurrently (as all index.ios* maps will be stored at `.vscode/.react/index.bundle.ios`. I experimented with replacing the path separator `/`with `_` but then the IDE didn't match anything. That's the part where I don't have a clue how that works. How does vscode map the source code to anything stored under `.vscode/.react`?

But - anyway. Just for the moment this will be an improvement (@digeff merge pls :-) ). I'm willing to drive this to the next level - in another PR, hopefully with your assistance.